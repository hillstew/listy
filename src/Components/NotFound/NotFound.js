import React from 'react';

const NotFound = () => {
    return (
        <h4>404 Page Not Found</h4>
    )
}

export default NotFound;