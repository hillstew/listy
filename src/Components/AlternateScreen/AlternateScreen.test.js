import React from 'react'
import NotFound from './NotFound';
import { shallow } from 'enzyme'

describe('NotFound', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<NotFound />)
    });

    it('should match the correct snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    });
});
