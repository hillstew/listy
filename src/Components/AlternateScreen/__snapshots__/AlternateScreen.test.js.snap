// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlternateScreen should match the correct snapshot 1`] = `
<div
  className="container-404"
>
  <h4
    className="screen-text"
  >
    <i
      class="fas fa-exclamation-triangle"
    />
    <p
      className="text-404"
    >
      404 page not found
    </p>
  </h4>
</div>
`;
